### ISSUE TITLE
AK/SK not yet implemented in 1.1.0

### ISSUE THREAD

**Issue by osaluden** [terraform-providers/terraform-provider-flexibleengine#37]
Hi there,

README.md shows how to configure the Flexible Engine provider whit the AccessKey/SecretKet wehereas this feature is not yet in GA.

### Terraform Version
```shell
$ terraform -v
Terraform v0.11.8
+ provider.flexibleengine v1.1.0
+ provider.openstack v1.9.0
```

### Affected Resource(s)
Please list the resources as a list, for example:
- provider.flexibleengine

### Terraform Configuration Files

provider.tf

```hcl
provider "flexibleengine" {
  access_key = "MY_ACCESS_KEY"
  secret_key = "MY_SECRET_KEY"
  auth_url    = "https://iam.eu-west-0.prod-cloud-ocb.orange-business.com/v3"
  region    = "eu-west-0"
}
```

### Output
```shell
$ terraform plan
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.


------------------------------------------------------------------------

Error: Error running plan: 1 error(s) occurred:

* provider.flexibleengine: You must provide a password to authenticate
```

### Expected Behavior
Plan should output the resources to be created/deteted/modified.

### Actual Behavior
Terraform ask for a user/password.

### Steps to Reproduce

1. Create provider.tf with the above content and your AK/SK.
2. export TF_LOG=DEBUG
3. `terraform plan`

**Comment by niuzhenguo** [comment]
@osaluden Thanks for raising this. That will be included in the coming release 1.2.0. We will try to make it released by the end of this week.

**Comment by freesky-edward** [comment]
@osaluden @niuzhenguo  yeah，the PR already merged #35 , we should watch the acc test results for some while

**Comment by aphentik** [comment]
Hello @niuzhenguo @freesky-edward , any news regarding the 1.2.0 release ? 

Did you also plan to provide a way to generate temporary AK/SK (STS) using terraform ?

**Comment by niuzhenguo** [comment]
@aphentik We will cut the 1.2.0 release by the end of this month. Sorry for the delay. 

But you can build the provider yourself with master code, please checkout the README.

**Comment by Julien-Le-Huludut-Orange** [comment]
Hello

I confirm this is solved in the 1.2 release, I tested it today.

**Comment by niuzhenguo** [comment]
@Julien-Le-Huludut-Orange thanks for the confirmation, so we can close this now.
